drivers/gles3/storage/particles_storage.cpp

```diff
@@ -341,12 +341,12 @@ void ParticlesStorage::particles_restart(RID p_particles) {
 
 void ParticlesStorage::particles_set_subemitter(RID p_particles, RID p_subemitter_particles) {
 	if (p_subemitter_particles.is_valid()) {
-		WARN_PRINT_ONCE("The OpenGL 3 renderer does not support particle sub emitters");
+		WARN_PRINT_ONCE_ED("The GL Compatibility rendering backend does not support particle sub emitters");
```

Comment:
Stylistic change for consistency with other messages:

```suggestion
		WARN_PRINT_ONCE_ED("The GL Compatibility rendering backend does not support particle subemitters.");
```

Also, I think we use "subemitter" in the documentation rather than "sub emitter" or "sub-emitter".